src/frontend/src/sass/_buttons.scss

```
@@ -101,7 +101,7 @@ a:hover {
   --text-color: white;
 
   border: 2px solid $bg-disabled;
-  cursor: not-allowed;
+  cursor: not-allowed !important;
   text-decoration: none;
```

I see text decoration is taking other properties as well. Try adding "important" in dev and observe the difference.

```suggestion
  text-decoration: none !important;
```